docs/search/search-functions.md

```diff
@@ -272,7 +272,7 @@ The phrase can be expressed as an arbitrary number of _phraseParts_ separated by
 - **skipTokens** (number, _optional_): amount of words/tokens to treat
```

Comment:
Why is optional italicized? The formatting through most of these is odd.

Reply:
@diana-macrometa This PR didn't edit Search Functions, that's another PR I am opening today.  This PR just changes the links in Search Functions to point to the appropriate topics.